lib/github_service/commands/cross_repo_test.rb

```diff
@@ -187,18 +187,31 @@ def dedup_repo_list(repo_list)
           .sort # Unadorned repo name will always sort before adorned repo names
           .uniq
           .slice_when { |a, b| bare_repo_name(a) != bare_repo_name(b) }
-          .map(&:last)
+          .map do |repos|
+            repos.shift if repos.size > 1 && bare_repo_name?(repos.first)
+            repos.size == 1 ? repos.first : repos
+          end
```

Comment:
Previously this just took the last one even if there were conflicts.  Now it will allow the conflicts to bubble out, so that `.valid?` can inspect them.  Note that a `[repo, repo#PR]` is not conflict...we prioritize the PR, which is why we drop the bare repo.